charts/orchestrator-k8s/hack/update_plugins_version.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -x
+
+# Get the directory of the script
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+TEMPLATE_FILE="${SCRIPT_DIR}"/../values.yaml "${SCRIPT_DIR}/../../../hack/update_plugins_version.sh"
+
```

Comment:
I don't think this will work in this case, since recently the OCP chart was changed, and pulled plugins version to the values.yaml to allow overriding them with staging/production values.

Reply:
I did exactly that  - I set the config in the values.yaml and the script works

Reply:
There was a change merged yesterday that changed its behavior - https://github.com/parodos-dev/orchestrator-helm-chart/pull/137/files#diff-32177824ed4ca4470ad62f26808a1e88a4ded33d25a102467159d703ca94fd60R3

that's why I suggest taking the previous version of the script and storing it here instead.
in this chart we don't care about plugins from staging/prod registries.

Reply:
ack